extensions/Worker.Extensions.ServiceBus/release_notes.md

```diff
@@ -4,6 +4,6 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.ServiceBus <version>
+### Microsoft.Azure.Functions.Worker.Extensions.ServiceBus 5.7.1
```

Comment:
We're pretty behind on the service bus releases, I would just bump this into a WebJobs version bump

https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.ServiceBus

The latest is 5.11.0. So I don't know if we want to do a release for 5.8, 5.8, 5.10 first or skip to 5.11?